src/pages/accounts/EmailSent.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import { toast } from 'react-toastify';
+const img =
+  'https://res.cloudinary.com/feyton/image/upload/v1645616111/Codebandits/Phantom_icon_1_tpjkws.png';
+
+function EmailSent() {
+  const { uuid, token } = useParams();
+  const [notified, setNotified] = useState(false);
+  const handleNotify = () => {
+    if (notified) {
+      toast('Already notified');
+    } else {
+      setNotified(true);
+      toast('Your issue was logged');
+    }
+  };
+
+  return (
+    <>
+      <h3 className="text-3xl text-center font-bold">Email Sent</h3>
+      <div className="text-left">
+        <p className="mb-1">
+          A password reset link has been sent to <b>"fab@me.com"</b>
+        </p>
+        <p className="mb-1">
+          Remember to check <b>Junk/Spam</b> folder.
```

Comment:
- This statement can come after Did not receive an email.(It's a reminder).
- Also you can work on the font size and alignment.